Directory.Build.props

```diff
@@ -37,7 +37,7 @@
 
   <!-- Versioning -->
   <PropertyGroup>
-    <ToolkitMinorVersion>4</ToolkitMinorVersion>
+    <ToolkitMinorVersion>5</ToolkitMinorVersion>
```

Comment:
In the past I've just bumped the `ToolkitPatchVersion`, rather than the minor.

"Minor version" bumps are reserved for either new integrations or Aspire releases, whereas bugfixes/etc. go via patch versions (essentially we freeze the major with Aspire and then don't really follow semver)